### PR TITLE
Parse and visualise security scheme. Closes #69

### LINKED ISSUE #69: Visualise security scheme
Visualise security scheme, based on `securityDefinitions` on open API spec.

For example, given
```
"securityDefinitions": {
    "Session": {
      "type": "apiKey",
      "name": "Session",
      "description": "Requests made to the Temando Platform API must be authenticated with a Session header whose value is a valid session token. Session tokens expire and must be recreated for further requests to be made of the Temando Platform API.",
      "in": "header"
    }
  },
```

The app could visualise the above information as below

image

### PR DESCRIPTION
**Pull request by brendo**
- parser can now handle `components.securitySchemes`, which will add `security` definitions to the parsed definition
- `SecurityContainer` component added to visualise security schemes at an API level (not per method!)
- minor CSS tweaks (I tried to restrain!) such as abstracting `.inline-pairs` so the CSS can be reused for other components. (I didn't want to create a `<dl>` component, it just felt like overkill.)

#115 has been raised for visualising the security scheme per method.

### CODE REVIEW

File: src/components/SecurityContainer/SecurityContainer.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+export default class SecurityContainer extends PureComponent {
+  render () {
+    const { id, security } = this.props
+    const isSimple = ['apiKey', 'http'].includes(security.type)
+
+    return (
+      <section className='security-container' id={id}>
+        {isSimple && this.renderSimple(security)}
+        {security.type === 'oauth2' && this.renderOAuth2(security)}
+        {security.type === 'openIdConnect' && this.renderOpenIdConnect(security)}
```
Comment (quangkhoa) [inline src/components/SecurityContainer/SecurityContainer.js]: Looks like all 3 render methods share something in common:
```
   <h3>{name} <code className='scheme'>type={type}</code></h3>
   {description && <Description description={description} />}
```

How about moving that code to here?

File: src/components/SecurityContainer/SecurityContainer.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    display: inline-block;
+    padding: .4rem;
+    margin-left: .5rem;
+    background-color: $navigation-background;
+  }
+
+  .flow-type {
+    padding: 1rem 0;
+
+    & + .flow-type {
```
Comment (quangkhoa) [inline src/components/SecurityContainer/SecurityContainer.scss:18]: When does this rule apply?

Comment (brendo) [reply src/components/SecurityContainer/SecurityContainer.scss:18]: When there are multiple flows provided, which is not possible with our current fixtures, but is possible with the spec.

File: src/components/SecurityContainer/SecurityContainer.js (hunk 1 above)
Comment (brendo) [reply src/components/SecurityContainer/SecurityContainer.js]: Good idea :+1:

File: src/parser/open-api/v3/securityParser.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/**
+ * Extracts any security schemes from the given Open API V3 definition file.
+ * Security schemes are defined in `components.securitySchemes`.
+ * The API defines what security applies at either the top level (`security`)
+ * or per operation.
+ *
+ * @param {Object} definition
+ * @return {Object}
+ */
+export function getSecurity (definition) {
```
Comment (quangkhoa) [inline src/parser/open-api/v3/securityParser.js]: This method is called `getSecurity`, but it returns `securityDefinitions`. This could be confusing.

**Pull request merged**